source/extensions/common/wasm/v8/v8.cc

```diff
@@ -385,7 +385,7 @@ absl::string_view V8::getPrecompiledSectionName() {
   return "";
 #else
   static const auto name =
-      absl::StrCat("precompiled_v8_v", V8_MAJOR_VERSION, ".", V8_MINOR_VERSION, ".",
+      absl::StrCat("precompiled_wee8_v", V8_MAJOR_VERSION, ".", V8_MINOR_VERSION, ".",
```

Comment:
This doesn't match what's generated by upstreamed `test/tools/wee8_compile/wee8_compile.cc`, which uses `precompiled_v8` prefix... (but that's not the reason for failing test, since the updated `.wasm` module also uses `precompiled_wee8` prefix)

Reply:
This is what envoyproxy/envoy-wasm has 👍 
207:  const std::string section_name = "precompiled_wee8_v" + std::to_string(V8_MAJOR_VERSION) + "." +

Reply:
Yes, but it was changed during upstreaming, and `envoyproxy/envoy` uses `precompiled_v8_v` prefix (this was change because `wee8` is an internal detail, and `v8` is a public name, so we should stick to it).

I don't care too much about one prefix or the other, but you've changed it only in the proxy, and not in the `wee8_compile` compile tool, so now they don't match, and pre-compiled Wasm modules will be never processed by the proxy.

Reply:
Changed it back to _v8_